.github/workflows/unittests.yml

```diff
@@ -116,24 +53,31 @@ jobs:
         run: ./bazelisk run larq_compute_engine/tflite/tests:cc_tests_aarch64_qemu --config=aarch64
 
   MLIR:
-    runs-on: ubuntu-latest
-    container:
-      image: tensorflow/tensorflow:custom-op-ubuntu16
-
+    runs-on: macos-latest
```

Comment:
I did some speed testing and by default the MLIR build took [42:39 min](https://github.com/larq/compute-engine/runs/441958454). Without the container, without the manylinux2010 toolchain and using `--distinct_host_configuration=false`, I can get it down to [31:32 min](https://github.com/larq/compute-engine/runs/443167125?check_suite_focus=true). Building using macOS is the fastest though with [19:24 min](https://github.com/larq/compute-engine/runs/443064715?check_suite_focus=true).